src/helpers/base-provider.js

```diff
@@ -13,8 +13,10 @@ import type {BaseProviderPropsT} from './types.js';
 const BaseProvider = (props: BaseProviderPropsT) => {
   const {children, overrides, theme, zIndex} = props;
   return (
-    <LayersManager zIndex={zIndex} overrides={overrides}>
-      <ThemeProvider theme={theme}>{children}</ThemeProvider>
+    <LayersManager zIndex={zIndex}>
+      <ThemeProvider theme={theme} overrides={overrides}>
```

Comment:
This is an API change we'll have to document right? Previously the overrides were for the LayersManager?